spynl/main/mail.py

```diff
@@ -90,6 +91,11 @@ def _sendmail(request, recipient, subject, plain_body, html_body=None,
     extra_headers = {}
     if reply_to:
         extra_headers['Reply-To'] = reply_to
+    if sender_name:
+        # sanitize sender name to make sure we do not get bad headers or
+        # unverified email errors.
+        sender_name = re.sub('[\n<>@]', ' ', sender_name).strip()
+        extra_headers['From'] = '{} <{}>'.format(sender_name, sender)
```

Comment:
I am not sure if it is useful as we substitute without checking but pyramid_mailer has a method [is_bad_headers](https://docs.pylonsproject.org/projects/pyramid_mailer/en/latest/_modules/pyramid_mailer/message.html#Message.is_bad_headers)